### DISCUSSION TITLE
Monthly gem5 Developers' Meeting - May 9th at 4pm (UTC)

### DISCUSSION THREAD

**Discussion by ivanaamit** [gem5/gem5#1095] [Announcements]
The monthly gem5 Developers' Meeting was held on May 9th.

### Reminder for the upcoming release of gem5 version 24.0: 
Creating the staging branch: May 20th (if you're working on something you want to include in the release, the change needs to be merged by this date)

Release date: June 3rd

**Comment by jmacneal** [comment]
Where can I find a link to the meeting (zoom/otherwise)?

**Reply by ivanaamit** [reply]
It will be here a few days before the meeting.
